src/template/commands/time.rs

```diff
@@ -0,0 +1,45 @@
+use std::collections::HashSet;
+
+use crate::template::run_multi::run_multi;
+use crate::template::timings::Timings;
+use crate::template::{all_days, readme_benchmarks, Day};
+
+pub fn handle(day: Option<Day>, force: bool) {
+    let stored_timings = Timings::read_from_file();
+
+    let mut days_to_run = HashSet::new();
+
+    match day {
+        Some(day) => {
+            days_to_run.insert(day);
+        }
+        None => {
+            all_days().for_each(|day| {
+                // when the force flag is not set, filter out days that are fully benched.
+                if force
+                    || !stored_timings
+                        .data
+                        .iter()
+                        .any(|t| t.day == day && t.part_1.is_some() && t.part_2.is_some())
```

Comment:
Maybe this part can be encapsuled in a method of `Timings` for readability like `Timings::is_day_complete(&self, Day) -> bool`.

Or have `Timings::get(&self, Day) -> Option<&Timing>` to get a day and `Timing::is_complete(&self) -> bool` to check if both parts are define.